.storybook/manager.js

```diff
@@ -1,11 +1,5 @@
-import favicon from '../stories/assets/icons/favicon.ico';
-import { addons } from '@storybook/addons';
-import theme from './theme';
-
-const link = document.createElement('link');
```

Comment:
This part adds a favicon: that's the only solution to add one I'm aware of.

But adding it back leads to a breaking change with the `No loader is configured for ".ico" files: stories/assets/icons/favicon.ico
` error.